R/docs/MetadataApi.md

```diff
@@ -0,0 +1,343 @@
+# MetadataApi
```

Comment:
Not sure if this will be a problem, but the actual file is `DataSetMetadataApi`, not `MetadataApi`.

Reply:
Won't affect things, but you're right that we should keep all names perfectly synced